code/game/verbs/who.dm

```diff
@@ -103,9 +110,18 @@
 		else if (R_DEBUG & C.holder.rights)
 			devwho += "\t[C] is a [C.holder.rank][extra]\n"
 			dev_count++
-
-	to_chat(src, "<b><big>Online staff:</big></b>")
-	to_chat(src, "<b>Current Admins ([admin_count]):</b><br>[adminwho]<br>")
-	to_chat(src, "<b>Current Moderators ([mod_count]):</b><br>[modwho]<br>")
-	to_chat(src, "<b>Current Mentors ([ment_count]):</b><br>[mentwho]<br>")
-	to_chat(src, "<b>Current Developers ([dev_count]):</b><br>[devwho]<br>")
\ No newline at end of file
+	var/msg = ""
+	if(!admin_count && !mod_count && !ment_count && !dev_count)
+		msg += "<b><big>Online staff:</big></b>"
+		if(!admin_count)
+			msg += "<b>Current Admins ([admin_count]):</b><br>[adminwho]<br>"
+		if(!mod_count)
+			msg += "<b>Current Moderators ([mod_count]):</b><br>[modwho]<br>"
+		if(!ment_count)
+			msg += "<b>Current Mentors ([ment_count]):</b><br>[mentwho]<br>"
+		if(!dev_count)
+			msg += "<b>Current Developers ([dev_count]):</b><br>[devwho]<br>"
+	if(!msg)
```

Comment:
Wouldn't this make the msg never display?

Reply:
Message will always be empty because check above uses `if (!var1 && !var2)` instead of `if(var1 || var2)`

Reply:
> Wouldn't this make the msg never display?

We don't need show null lists Admins